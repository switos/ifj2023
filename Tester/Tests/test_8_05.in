let sum = a + b
// If 'a' and 'b' do not have specified types
