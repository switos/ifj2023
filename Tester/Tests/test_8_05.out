Expected Output:

Expected Return Code: 5
